packages/block-library/src/media-text/index.js

```diff
@@ -82,6 +83,67 @@ export const settings = {
 		align: [ 'wide', 'full' ],
 	},
 
+	transforms: {
+		from: [
+			{
+				type: 'block',
+				blocks: [ 'core/image' ],
+				transform: ( { align, alt, url, id } ) => (
+					createBlock( 'core/media-text', {
+						align,
+						mediaAlt: alt,
+						mediaId: id,
+						mediaUrl: url,
+						mediaType: 'image',
+					} )
+				),
+			},
+			{
+				type: 'block',
+				blocks: [ 'core/video' ],
+				transform: ( { src, align, id } ) => (
+					createBlock( 'core/media-text', {
+						align,
+						mediaId: id,
+						mediaUrl: src,
+						mediaType: 'video',
+					} )
+				),
+			},
+		],
+		to: [
```

Comment:
When transforming from a media-text to a video or image I noticed the block retains its wide format instead of shrinking down to the size of a normal image block. Is that the desired behaviour?

Reply:
Hi @talldan now we are using the default aligns of the block during transforms, the problem should be fixed.